schema/me/conversation/index.js

```diff
@@ -252,14 +271,12 @@ export default {
   },
   resolve: (root, { id }, request, { rootValue: { accessToken } }) => {
     if (!accessToken) return null
-    return gravity
-      .with(accessToken, { method: "POST" })("me/token", {
-        client_application_id: IMPULSE_APPLICATION_ID,
-      })
-      .then(data => {
-        return impulse.with(data.token, { method: "GET" })(`conversations/${id}`).then(impulseData => {
-          return impulseData
-        })
+    return gravity.with(accessToken, { method: "POST" })("me/token", {
+      client_application_id: IMPULSE_APPLICATION_ID,
+    }).then(data => {
+      return impulse.with(data.token, { method: "GET" })(`conversations/${id}`).then(impulseData => {
+        return impulseData
```

Comment:
I know you didn’t write this (at least not in this PR), but this return statement is superfluous, just returning the promise is enough.